README.md

```diff
@@ -1,4 +1,5 @@
-![CI](https://github.com/eclipse/che-dashboard/workflows/CI/badge.svg)
+[![CI](https://github.com/eclipse/che-dashboard/workflows/CI/badge.svg)](https://github.com/eclipse/che-dashboard/actions/workflows/ci.yaml)
+[![Codecov](https://img.shields.io/codecov/c/github/eclipse/che-dashboard)](https://app.codecov.io/gh/eclipse/che-dashboard)
```

Comment:
Don't we need to specify the specific branch?
```suggestion
[![Codecov](https://img.shields.io/codecov/c/github/eclipse/che-dashboard/master)](https://app.codecov.io/gh/eclipse/che-dashboard)
```

Reply:
I'm pretty sure that without specifying a branch it uses a default one.